src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts

```diff
@@ -7,12 +13,69 @@ import { Component, OnInit } from '@angular/core';
 export class CertificateDetailsComponent implements OnInit {
   showSuccessModal: boolean;
   loader: boolean;
-  error: boolean;
   viewCertificate: boolean;
-  viewCertificateHtml: boolean;
+  error: boolean;
+  enableVerifyButton = false;
+  certificateCode: any;
+  wrongCertificateCode = false;
+  instance: string;
+  watchVideoLink: SafeResourceUrl;
+
+  @ViewChild('codeInputField') codeInputField: ElementRef;
+
+  constructor(
+    public activatedRoute: ActivatedRoute,
+    public certificateService: CertificateService,
+    public resourceService: ResourceService,
+    public router: Router,
+    private sanitizer: DomSanitizer
+  ) { }
 
-  constructor() { }
   ngOnInit() {
+    this.instance = _.upperCase(this.resourceService.instance);
+    this.watchVideoLink = this.sanitizer.bypassSecurityTrustResourceUrl('https://www.youtube.com/embed/2Xgff-gsf0A?rel=0&autoplay=1');
+  }
+
+  verify(event: any) {
```

Comment:
please make the method name as certificateVerify

Reply:
done